src/util/formDevices.tsx

```diff
@@ -29,10 +35,19 @@ export interface CustomNetworkDevice {
   bare: LxdNicDevice;
 }
 
+export type FormDiskDevice = Partial<LxdDiskDevice> &
+  Required<Pick<LxdDiskDevice, "name" | "pool">> & {
+    limits?: {
+      read?: string;
+      write?: string;
+    };
```

Comment:
With the change proposed above, this should have now become part of the `LxdDiskDevice` type which is already expressed by `Partial<LxdDiskDevice>`, so I think it could be removed from here.

Reply:
We have to keep the structure for LXD different, but as formik doesn't allow keys to contain a `.`, we use this different structure for the form type.